crates/astria-sequencer-relayer/src/relayer/mod.rs

```diff
@@ -276,14 +276,19 @@ impl Relayer {
     }
 
     #[instrument(skip_all, fields(%height))]
+    #[expect(
```

Comment:
I mean, you could just give this an alias, `type ForwardingFut = Fuse<BoxFuture<...>>`